1.0/docs/migration.md

```diff
@@ -340,7 +340,7 @@ for more information.
 
 The value is interpreted as a method name to be invoked when the property value 
 changes. Note that unlike in 0.5, **property change handlers must be registered 
-explicitly.** The <code><var>propertyName</var>-changed</code> method will not be 
+explicitly.** The <code><var>property-name</var>-changed</code> method will not be 
```

Comment:
This line should stay as it is, since it's talking about the old 0.5 change watchers:

https://www.polymer-project.org/0.5/docs/polymer/polymer.html#change-watchers